icu4c/source/i18n/ucal.cpp

```diff
@@ -58,12 +58,12 @@ ucal_openTimeZoneIDEnumeration(USystemTimeZoneType zoneType, const char* region,
 
 U_CAPI UEnumeration* U_EXPORT2
 ucal_openTimeZones(UErrorCode* ec) {
-    return uenum_openFromStringEnumeration(TimeZone::createEnumeration(), ec);
+    return ucal_openTimeZoneIDEnumeration(UCAL_ZONE_TYPE_ANY, NULL, NULL, ec);
```

Comment:
Would you mind amending your commit to using nullptr instead of NULL? (Both functions you are changing.)

Reply:
ok - will do.